config/initializers/cors.rb

```diff
@@ -8,7 +8,10 @@
 
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
-    origins Rails.application.config.allowed_cors_origins
+    origins Rails.application.config.allowed_cors_origins,
+      /\Ahttp:\/\/192\.168\.\d+\.\d+(:\d+)?\z/,
+      /\Ahttp:\/\/127\.0\.0\.1(:\d+)?\z/,
+      /\Ahttp:\/\/localhost(:\d+)?\z/
```

Comment:
Rather than appending this to the initializer, you should update the allowed_cors_origins config values under config.